src/cli.js

```diff
@@ -1,36 +1,45 @@
 const { Command } = require('commander');
 const fs = require('fs');
 const path = require('path');
-const exportModule = require('./exportModule.js');
-const logger = require('./helpers/winston.js');
+const exportModule = require('./exportModule');
+const logger = require('./helpers/logger');
 
 const program = new Command();
 program.version('0.0.1');
-function data() {
+function loadELM() {
   program
+    .version('0.0.1')
     .requiredOption('--elmJSON <file>', 'elm json to turn to synthea module')
-    .option('--library <library>', 'directory of dependent libraries')
+    .option('--dependency <library>', 'directory of dependent libraries')
     .parse(process.argv);
-  const returns = {};
-  const mainFile = fs.readFileSync(program.elmJSON, 'utf8');
-  returns.mainLibrary = JSON.parse(mainFile);
-  if (program.library !== undefined) {
-    const directoryPath = path.join(__dirname, program.library);
-    returns.dependencies = [];
+  const ELMFiles = {};
+  try {
+    const mainFile = fs.readFileSync(program.elmJSON, 'utf8');
+    ELMFiles.mainLibrary = JSON.parse(mainFile);
+  } catch (err) {
+    logger.error(err.message);
+    process.exit(1);
+  }
+  if (program.dependency !== undefined) {
+    const directoryPath = path.resolve(program.dependency, './');
+    ELMFiles.dependencies = [];
+    const dependentFiles = fs.readdirSync(directoryPath).filter((file) => {
+      return file.split('.').pop() === 'json' ? file : null;
```

Comment:
the return result of `filter` should just be a boolean. Technically this is working, but not for the reasons you expect. Yay javascript!

```suggestion
      return file.split('.').pop() === 'json';
```